docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Configure a Spring Boot Initializer app to use Azure Application Insights SpringBoot Starter
+description: Configure a Spring Boot application created with the Spring Initializr to use Appliaction Insights SpringBoot Starter.
+services: Application-Insights
+documentationcenter: java
+author: dhdoshi
```

Comment:
Change author: to your GitHub login. (This gets used by some of the automated notifications/reporting)

author: takes a single value consisting of github login
ms.author: is fine as is and takes multiple Microsoft aliases as values separated by semicolon